3rdparty/test_git_modules.py

```
@@ -0,0 +1,105 @@
+]
+
+ThirdpartyViolation = collections.namedtuple(
+    "ThirdpartyViolation", ["section", "path", "note"]
+)
+
+
+def find_violations(config: configparser.ConfigParser) -> list[str]:
+    violations = []
+    for section in config.sections():
+        if not section.startswith("submodule "):
+            raise ValueError(f"Unexpected section in .gitmodules: {section}")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)